### PR TITLE
polish (davide): Polish config davide

### PR DESCRIPTION
**Pull request by davide97l**
## Description

Polish config of Bsuite and Mujoco (Ant)

### CODE REVIEW

File: dizoo/mujoco/config/ant_ppo_config.py [hunk 1]
```diff
@@ -19,7 +19,6 @@
         model=dict(
             obs_shape=111,
             action_shape=8,
-            continuous=True,
```
Comment (PaParaZz1) [inline dizoo/mujoco/config/ant_ppo_config.py:22]: why remove `continuous` here

File: dizoo/mujoco/config/ant_onppo_config.py [hunk 2]
```diff
@@ -35,17 +32,12 @@
...
-            # for halfcheetah, the length=1000
-            # ignore_done=True,
-            ignore_done=False,
+            ignore_done=False,  # when we recompute advantage, we need the key done in data to split trajectories
             grad_clip_type='clip_norm',
             grad_clip_value=0.5,
         ),
         collect=dict(
-            collector_env_num=collector_env_num,
+            collector_env_num=10,
```
Comment (PaParaZz1) [inline dizoo/mujoco/config/ant_onppo_config.py:40]: `collector_env_num` should be unused here

File: dizoo/mujoco/config/ant_onppo_config.py [hunk 3]
```diff
@@ -35,17 +32,12 @@
             clip_ratio=0.2,
             adv_norm=True,
             value_norm=True,
-            # for onppo, when we recompute adv, we need the key done in data to split traj, so we must
-            # use ignore_done=False here,
-            # but when we add key traj_flag in data as the backup for key done, we could choose to use ignore_done=True
-            # for halfcheetah, the length=1000
-            # ignore_done=True,
-            ignore_done=False,
+            ignore_done=False,  # when we recompute advantage, we need the key done in data to split trajectories
```
Comment (PaParaZz1) [inline dizoo/mujoco/config/ant_onppo_config.py:35]: `traj_flag` is the effect you commented, `ignore_done` has other mechanism

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: dizoo/mujoco/config/ant_ppo_config.py (hunk 1 above)
Comment (davide97l) [reply dizoo/mujoco/config/ant_ppo_config.py:22]: changed 'continuous=True' with 'action_space='continuous'' in all ppo/onppo/trex configs in mujoco

File: dizoo/mujoco/config/ant_onppo_config.py (hunk 2 above)
Comment (davide97l) [reply dizoo/mujoco/config/ant_onppo_config.py:40]: solved

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**